cfme/containers/provider/openshift.py

```diff
@@ -55,6 +55,25 @@ def view_value_mapping(self):
         return out
 
 
+class AlertsEndpoint(DefaultEndpoint):
+    """Represents Alerts Endpoint"""
+    name = 'alerts'
+
+    @property
+    def view_value_mapping(self):
+        if current_version() > '5.9':
```

Comment:
**REQUIRED**
`current_version()` is deprecated, please use `appliance.version` or `self.browser.product_version` in views.

Reply:
Done